presqt/exceptions/exceptions.py

```diff
@@ -0,0 +1,15 @@
+class PresQTException(Exception):
+    """
+    Custom exception that adds 'data' and 'status_code' to the exception so when the exception is
+    raised we can dynamically return the proper Response object.
+    """
+    def __init__(self, data, status_code):
```

Comment:
I *think* you might want to add *args, **kwargs and then call the __init__() method of Exception inside of this to make sure the normal exception functionality is still present. :)